e2e/transfer_test.go

```diff
@@ -114,6 +115,53 @@ func (s *TransferTestSuite) TestMsgTransfer_Succeeds_Nonincentivized() {
 	})
 }
 
+func (s *TransferTestSuite) TestMsgTransfer_Timeout_Nonincentivized() {
+	t := s.T()
+	ctx := context.TODO()
+
+	relayer, channelA := s.SetupChainsRelayerAndChannel(ctx, transferChannelOptions())
+	chainA, chainB := s.GetChains()
+	chainAWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+	chainBWallet := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
+
+	chainBWalletAmount := ibc.WalletAmount{
+		Address: chainBWallet.Bech32Address(chainB.Config().Bech32Prefix), // destination address
+		Denom:   chainA.Config().Denom,
+		Amount:  testvalues.IBCTransferAmount,
+	}
+
+	t.Run("IBC transfer packet timesout", func(t *testing.T) {
+		tx, err := chainA.SendIBCTransfer(ctx, channelA.ChannelID, chainAWallet.KeyName, chainBWalletAmount, testvalues.ImmediatelyTimeout())
+		s.Require().NoError(err)
+		s.Require().NoError(tx.Validate(), "source ibc transfer tx is invalid")
+		time.Sleep(time.Nanosecond * 1) // want it to timeout immediately
```

Comment:
same difference I think

```suggestion
		time.Sleep(time.Nanosecond) // want it to timeout immediately
```